### ISSUE TITLE
[Content Area] Get Started with PostCSS

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.org#110]
There is a content area right after #109 with two buttons to get started with PostCSS. We would need to apply some button markup and styling for these buttons (primary, secondary btns) 

You can find all the necessary styleguide info from Zeplin - #98

![screenshot-get-started](https://cloud.githubusercontent.com/assets/2952332/12494845/c73d00a8-c08c-11e5-9e2c-fc0d875e58fa.png)

**Comment by jeddy3** [comment]
FYI, I called this component `WayFinder` in `1.0`

**Comment by marcustisater** [comment]
Yeah, we replaced this with the other buttons - it's kind of like a site map shortcuts. It's super nice to have.

Good work
